README.md

```diff
@@ -1,91 +1,7 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Two Cent Stories
 
-## Available Scripts
+The goal of this project is to demonstrate how much we can accomplish with [Elizabeth Warren's Two-Cent Tax](https://elizabethwarren.com/plans/ultra-millionaire-tax). Tell us about your experience with student debt, and see how easily it can be eliminated with the two-cent tax!
 
-In the project directory, you can run:
+## Working version
 
-### `npm start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
```

Comment:
I suspect some of these are real commands. Perhaps `npm run build` in particular